src/gui.py

```diff
@@ -568,6 +577,7 @@ def sliceSTL(self, slicing_type):
             "print_speed_layer1": self.printSpeedLayer1_value.text(),
             "print_speed_wall": self.printSpeedWall_value.text(),
             "nozzle": self.nozzle_value.text(),
+            "filling_type": self.filling_type_values.currentIndex(),
```

Comment:
i prefer sending text values, it is easier for debug.

Reply:
If will increase quantity of code and decreasce quality because we should check locale and translate into English. In that case i think that your recomendation will be a crutch, isn't it?